src/adv_routines.F

```diff
@@ -3384,8 +3384,7 @@ subroutine vadv_flx3(stag,ix,jy,kz,c1,c2,rrw,dumz,a)
 
   IF( stag.eq.1 )THEN
 
-!$omp parallel do default(shared)   &
-!$omp private(i,j,k),
+    !$omp parallel do default(shared) private(i,j,k)
```

Comment:
why is this a syntax error? It seems to me a stylish choice.

Reply:
@sjsprecious, the presence of the comma after the priviate(i,j,k) clause is a syntax error

Reply:
Oh, thanks for pointing that out. Sorry that I did not catch it!